src/main/flight/pid.c

```diff
@@ -331,6 +340,35 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     itermRotation = pidProfile->iterm_rotation == 1;
 }
 
+#ifdef USE_TXPID
+void pidUpdateRates(pidProfile_t *pidProfile, int16_t *rcData)
+{
+    int16_t stickCenter = flight3DConfig()->neutral3d;
```

Comment:
This is not the stick center. `PWM_RANGE_MIDDLE`

Reply:
Thanks. That's much neater.